### ISSUE TITLE
gui/launcher does not work

### ISSUE THREAD

**Issue by KonstantinBerkow** [DFHack/dfhack#2902]
When trying to open gui/laucher in fortress mode via hotkeys (Ctrl+Shift+P, Ctrl+Shift+D) or via mouse nothing happens.
In DFHack console I get single error line:
![image](https://user-images.githubusercontent.com/8669019/218537703-eda4b112-4641-4d90-8bb1-a6596e6a3b2b.png)

Per @myk002 advice I've checked following files: `dfhack-config/launcher.json`, `dfhack-config/command_counts.json` and `hack/data/base_command_counts.json`

Looks like launcher and command_counts files were corrupted, or generation failed.
When I deleted this two files `gui/launcher` seems to work properly, after running some commands this files were regenerated and now can be viewed via VSCode (it wasn't possible to view old files).

I can't attach old files - github does not allow their file type.

**Comment by lethosor** [comment]
Could you zip the old files and upload that?

**Comment by KonstantinBerkow** [comment]
@lethosor 
[dfhack_gui_launcher_bug_report.zip](https://github.com/DFHack/dfhack/files/10725241/dfhack_gui_launcher_bug_report.zip)

**Comment by lethosor** [comment]
Found the files on Discord, assuming they're the same:
* command_counts.json is 429 NULL bytes
* launcher.json is 44 NULL bytes

The only likely culprit I could think of on our end would be dangling file pointers, i.e. files not closed properly between reads/writes. But our `json.lua` library calls close() at all appropriate times (there is only one file I/O path for reading and one for writing, and they both call close()).

I think the most likely cause here is some sort of disk issue. Either your disk ran out of space, or it's experiencing some sort of corruption, or it lost power before the write could complete. It could also be an OS-level issue, like antivirus software preventing the writes.

In other words, I don't think there's much we can do here, sorry.

**Comment by KonstantinBerkow** [comment]
Thank you Alan, I've already found and added solution into issue description. Should be save to close.
